packages/components/src/components/pin-code/pin-code-utils.ts

```diff
@@ -0,0 +1,72 @@
+import type { FormState } from '../../utils/form/form-state';
+import type { Styles } from 'jss';
+import { setAttribute } from '../../utils';
+import { consoleWarn } from '../../utils/log';
+
+export const PIN_CODE_TYPES = ['number', 'password'] as const;
+export type PinCodeType = (typeof PIN_CODE_TYPES)[number];
+
+export const PIN_CODE_LENGTHS = [4 as number, 6 as number] as const;
```

Comment:
I think the number casting shouldn't be needed.

Reply:
![image](https://github.com/porsche-design-system/porsche-design-system/assets/109523972/9ab31253-70f7-4adf-86a9-a7354f105943)
Unfortunately it seems to be needed :(